recipes/ffbidx/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "1.0.1" %}
+
+{% set cuda_major = (cuda_compiler_version|default("12.0")).rpartition('.')[0] %}
+
+package:
```

Comment:
Please indent with two spaces like we do in all other recipes.

Reply:
Done